### ISSUE TITLE
Enable selectively ‘notifying producers’ in an OC

### ISSUE THREAD

**Issue by jaycmb** [openfoodfoundation/wishlist#100]
## What is the need / problem?

The original post identified 2 needs

a) Is it possible to select certain suppliers to be notified rather than the default - which is to notify all at present? For example, at the OC setup, could there be a notify - Y/N - selection beside each producer/supplier with product ‘incoming’ into the OC?

b) It is also a problem that there can only be one email address for notifications. In a scenario where hubs share a supplier, and are co-managing the product list (the supplier doesn’t want to do it) - there is no way to specify which hub manager gets notified in the OC? Am I missing something 

## Which type of users does this problem affect (and how many, if known)?
_- Hub Managers (MCFEs)_
_- Producers_

## Connected wishlist and discovery discussions

Migrated from Discourse: https://community.openfoodnetwork.org/t/is-it-possible-to-selectively-notify-producers-in-an-oc/983

Originally opened
When: Jun '17
By: tschumilas

**Other connected issues:**
Allow for >1 notification email address per enterprise: https://github.com/openfoodfoundation/openfoodnetwork/issues/1665

## Additional context

From sstead:

a) Regarding being able to notify individual producers, rather than all of them, Rob suggested that this would be relatively straightforward, perhaps half a day
b) Regarding being able to select which email the notification goes too, Rob said this would be fiddlier and take more time. He mentioned that the notify producers modal is quite small, so it may be hard to store all the functionality in a small space, especially for large OCs with many producers.

## Is Papercut or feature candidate?

Seems like a) could be done as a papercut

**Comment by jaycmb** [comment]
Papercut Meeting Sept 8th

More than a papercut.

- Needs more investigation for the real problem (why should only be a subset of producers notified)?
- we don´t want to patch something that might need to be "fixed at the base" (like why do we have the _notify producers_ button only on the first step of the OC process?)

**Comment by mariocarabotta** [comment]
**Discussed minimum version during delivery circle**

- add notify button in second step of order cycle
- add one button per producer that sends individual email

**Comment by RachL** [comment]
Final estimate : 2 dev days and 2 design days (that we can shorter to 0,5 if we reuse an existing button style)

**Comment by drummer83** [comment]
@RachL Is testing included in dev time? I would estimate 0.5 days.

**Comment by RachL** [comment]
@drummer83 nope. We currently multiply dev time by 6 to account for everything else (product, design, code review, testing). 

Another rule that work is to count for 20% of dev time, which is 0.5 days and matches your estimate. So perhaps going forward we need to especially make sure there is no particularity that would blow up that rule.
